.clang-tidy-diff-scans.txt

```diff
@@ -2,7 +2,10 @@
 ./iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/*
 ./iceoryx_hoofs/test/moduletests/test_cxx*
 ./iceoryx_hoofs/source/cxx/*
-
+./iceoryx_hoofs/source/relocatable_pointer/relative_pointer_data.cpp
+./iceoryx_hoofs/test/moduletests/test_relative_pointer_data.cpp
```

Comment:
This file is not changed by this PR. @mossmaurice did you perform a manual clang-tidy-15 scan?

Reply:
@elfenpiff I guess the reason is that it is a textbook example on how to write tests ... hmm, where does this stinky smell suddenly come from ... must be self-laudation :laughing: ... sorry, couldn't resist

Reply:
Before I approve: @mossmaurice did you perform a manual clang-tidy-15 scan?